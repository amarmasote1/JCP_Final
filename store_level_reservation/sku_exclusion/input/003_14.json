{
  "reservation": [{
    "storeID": "1037",
    "reservationID": "6df7f9dc-60b3-4f15-bd99-cb8498d6a222",
    "procurementAllowed": "false",
    "shippable": "true",
    "uom": "EACH",
    "itemID": "80110002004",
    "lineID": "3",
    "qty": "10"
  }
  ]
}