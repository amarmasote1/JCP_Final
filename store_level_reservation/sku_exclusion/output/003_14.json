{
    "message": "Success",
    "body": [
        {
            "partiallyReservedQty": 0,
            "shortageQty": 0,
            "orgCode": "JCP",
            "reservedQty": 10,
            "requestLine": {
                "deliveryMethod": null,
                "channel": null,
                "lineID": "3",
                "storeID": "1037",
                "itemID": "8011000-2004",
                "schedule": false,
                "uom": "EACH",
                "shippable": true,
                "reservationID": "6df7f9dc-60b3-4f15-bd99-cb8498d6a222",
                "procurementAllowed": false,
                "qty": 10,
                "bopus": false,
                "increaseCapacity": null,
                "shutdown": false
            },
            "problemType": "SUCCESS"
        }
    ],
    "status": 200
}