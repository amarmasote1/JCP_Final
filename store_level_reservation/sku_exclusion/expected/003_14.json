{"status":200,"message":"Success","body":[{"shortageQty":0,"reservedQty":10,"partiallyReservedQty":0,"problemType":"SUCCESS","orgCode":"JCP","requestLine":{"lineID":"3","itemID":"8011000-2004","uom":"EACH","bopus":false,"shippable":true,"deliveryMethod":null,"increaseCapacity":null,"shutdown":false,"qty":10,"storeID":"1037","channel":null,"reservationID":"6df7f9dc-60b3-4f15-bd99-cb8498d6a222","procurementAllowed":false}}]}